spec/spec_helper.rb

```diff
@@ -1,2 +1,11 @@
 require 'simplecov'
-SimpleCov.start
\ No newline at end of file
+SimpleCov.start
+
+require 'csv'
+require './lib/game'
+require './lib/team'
+require './lib/team_result'
+require './lib/game_parser'
+require './lib/league_parser'
+require './lib/team_result_parser'
+require './lib/stat_tracker'
```

Comment:
This looks great! Thanks for building this `spec_helper` out to help keep things more organized.